carbon-design-system/carbon
# fix(accordion): update arrow spacing for start variant

Currently, the chevrons in the "start"-variant of the Accordion (with the chevrons on the left)  have 8px of left spacing whereas the chevrons in the "end"-variant (with the chevrons on the right) have a right spacing of 16px.

After checking with @aagonzales, the left spacing on the "start"-variant should also be 16px / spacing-05.

Before
![image](https://user-images.githubusercontent.com/28265588/135447167-02e3251d-8a0e-4c78-82c1-98c315f3530b.png)

After
![image](https://user-images.githubusercontent.com/28265588/135447213-fb938c8f-66f8-4cb3-be13-230f3132447a.png)


#### Changelog

**Changed**

- Updated left spacing of chevron in "start"-variant to use `$spacing-05`/16px.
- Added a `$spacing-05`/16px spacing to the right of the accordion title based on spec from @aagonzales:
  - ![image](https://user-images.githubusercontent.com/28265588/135447345-8b8f4d1d-5535-434f-bf97-99faf6c8fb3d.png)

#### Testing / Reviewing

1. Run react storybook / open netlify preview
2. Ensure that the chevrons always have 16px to their left and right regardless of variant
3. Ensure the title has a right spacing of 16px in the "start"-variant
